airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -59,7 +59,7 @@ def chunk_date_range(
     """
     today = pendulum.today(tz=time_zone)
     end_date = min(pendulum.parse(end_date, tz=time_zone), today) if end_date else today
-    start_date = pendulum.parse(start_date, tz=time_zone)
+    start_date = pendulum.parse(start_date, tz=time_zone) if start_date else pendulum.now().subtract(years=2)
```

Comment:
If the start date is not specified, we ought to attempt to retrieve all available data. Alternatively, we can include a comment indicating that the date range query parameters are mandatory, prompting us to assign a default start date if it is absent.

Reply:
Date range in mandatory parameter for the query.